libportal/camera.c

```diff
@@ -47,7 +48,7 @@ xdp_portal_is_camera_present (XdpPortal *portal)
                                      "org.freedesktop.DBus.Properties",
                                      "Get",
                                      g_variant_new ("(ss)", "org.freedesktop.portal.Camera", "IsCameraPresent"),
-                                     G_VARIANT_TYPE_BOOLEAN,
+                                     G_VARIANT_TYPE ("(v)"),
```

Comment:
```suggestion
                                     G_VARIANT_TYPE_VARIANT,
```

Reply:
Isn't that different though `"(v)"` vs `"v"`?

Reply:
@nielsdg is correct. The `reply_type` for a method call in GDBus is always a tuple with zero or more types in parentheses, never just a plain type (because replies can "return" more than one "out" parameter).

In this case we're representing

```
o.fd.DBus.Properties.Get(s: interface, s: property) -> v: value
```

as

```
in parameters: (ss)
out parameters: (v)
```

Reply:
I see, thanks for clarifying